mathics/core/rules.py

```diff
@@ -126,10 +131,19 @@ def do_replace(self, expression, vars, options, evaluation):
         vars_noctx = dict(((strip_context(s), vars[s]) for s in vars))
         if self.pass_expression:
             vars_noctx["expression"] = expression
+        builtin_name = self.function.__qualname__.split(".")[0]
+        stat = function_stats[builtin_name]
+        ts = time.time()
+
+        stat["count"] += 1
         if options:
-            return self.function(evaluation=evaluation, options=options, **vars_noctx)
+            result = self.function(evaluation=evaluation, options=options, **vars_noctx)
         else:
-            return self.function(evaluation=evaluation, **vars_noctx)
+            result = self.function(evaluation=evaluation, **vars_noctx)
+        te = time.time()
+        elapsed = int((te - ts) * 1000)
```

Comment:
@rocky I'm just in doubt of the unit of seconds, look at [time](https://docs.python.org/3/library/time.html#time.time).

Reply:
The subtraction gives the time sections. The granularity is in milliseconds which is given in a floating point number less than zero. So we multiply by 1000 to get it to be an integer number of milliseconds.